README.md

```diff
@@ -12,26 +12,85 @@ $ mkdir "$HOME/data_server/results" -p
 
 ## Usage
 
-Use the host network, `--net=host`, for traffic to get from a remote client through the host to the container. The bind mount volume is labeled as a shared directory with the host, `:z`, to persist results and facilitate integration with other tools on the host.
+### Overview
+
+1. Create a means of communication between the containers.
+    * Podman containers can easily communicate with `localhost` when they're in the same pod.
+    * Docker containers can easily communicate over the default `bridge` network when each service is given a `network-alias`.
+
+2. Start a Postgresql database service.
+
+3. Edit the environment variables with appropriate usernames and passwords. Run the application deployment's `prestart` to seed the database with user accounts.
+
+4. Execute the deployment's `start` script to get the server running.
+
+### Notes
+
+* On my local machine, when I have more than a few environment variables I prefer to add all of  them to an `.env` file.
+
+* The bind mount volume is labeled as a shared directory with the host, `:z`, to persist results and facilitate integration with other tools on the host.
+
+### Common Variables
+
+```shell
+$ pgvol=pgvol
+$ db_name=pg-svc
+$ webserver_name=snap-web
+```
+
+### Podman
```

Comment:
Slight title change: "Running with Podman" or "Podman invocation" or something like that